### PR TITLE
fb303 fbthrift fizz folly wangle watchman 2022.08.15.00

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/watchman.rb [hunk 1]
```diff
@@ -2,19 +2,14 @@ class Watchman < Formula
...
+    sha256 "8b95b0ac3de81bd7bc7d243e4785d4e9f8e8c41d70a0fb8b03425c078bdcc434"
 
-    # `edencommon` currently doesn't provide tagged releases, so we just use a commit
-    # around release time of `watchman`. For example, we can use the `edencommon` commit
-    # that updates folly-rev.txt file to match `folly` formula version release commit.
-    # TODO: Once tags are available, either switch to tag or create a dedicated formula.
     resource "edencommon" do
-      url "https://github.com/facebookexperimental/edencommon/archive/d356bb7a9a28f09f00d72a81f7b60f8f27ce249c.tar.gz"
-      sha256 "d5c5277bb697f131abd591d52d8b9cb905309d516fd767a4d6cdf3b156659060"
+      url "https://github.com/facebookexperimental/edencommon/archive/refs/tags/v2022.08.15.00.tar.gz"
```
Comment (chenrui333) [inline Formula/watchman.rb:11]: switch to use tagged commit

**Comment by carlocab** [comment]
Pushed a fix at #105132. I want to test it before upstreaming.

We should probably also roll this into #105132 to avoid doubling rebuilds.

File: Formula/watchman.rb [hunk 2]
```diff
@@ -2,19 +2,14 @@ class Watchman < Formula
...
+      sha256 "5e04fc7a0577160f5736a411097086bd85eeef5066f44387b00305b107b0563a"
     end
```
Comment (carlocab) [inline Formula/watchman.rb:13]: Would be nicer to move this to a separate formula, since it's pretty easy to forget to update this.

**Comment by carlocab** [comment]
Merged in #105132.

**Pull request closed without merging**